src/game_state/GameObject.cpp

```diff
@@ -2,14 +2,15 @@
 
 #include "game_state/GameObject.h"
 
+#include <assert.h>
+
+#include "helpers/Logging.h"
```

Comment:
since we have
`include_directories(${CMAKE_SOURCE_DIR}/src/helpers)`
in the CMakeLists.txt you can just
`#include "Logging.h"`